openapi/openapi-service.ytt.yaml

```diff
@@ -127,12 +127,21 @@ paths:
       tags:
         - service
       responses:
-        '200':
+        "200":
           description: Healthz response
           content:
             text/plain:
               schema:
                 type: string
                 example: healthz check passed
-        '4XX':
+        "4XX":
           description: error
+
+  /issues:
```

Comment:
Did I get this right; we plan to remove this endpoint before release, and it is just here for testing purposes?

If so, we might not have to add this definition. Then, if we accidentally release with it, it won't be publicly exposed/documented.

Reply:
It is not just for testing, it is just marked as a beta feature to advise people to not depend on this too critically, since it might change or get added to `/telemetry` once it is stable enough.

I think it is useful to document it, to make the development of the webUI feature easier

Reply:
Check! In that case I agree, keep it here.